src/main/java/com/easypost/model/Tracker.java

```diff
@@ -148,7 +149,16 @@ public static boolean createList(Map<String, Object> params) throws EasyPostExce
 	public static boolean createList(Map<String, Object> params, String apiKey) throws EasyPostException {
 		String createListUrl = String.format("%s/create_list", classURL(Tracker.class));
 
-		request(RequestMethod.POST, createListUrl, params, Object.class, apiKey);
+		int count = 0;
+		Map<String, Object> newParams = new HashMap<String, Object>();
+		Map<String, Object> trackers = new HashMap<String, Object>();
+		for (Object tracker : (ArrayList)params.get("trackers")) {
+			trackers.put(String.valueOf(count), tracker);
+			count++;
```

Comment:
Why is it necessary to put `count`, are we not using JSON automatic serialization?

Also, this PR mentions adding tax identifiers, but this change is on `Tracker.java` -- should this change be broken out into a separate PR?

Reply:
Tracker requires a specific format that deviates from the normal format. We need to make a dictionary of number keys and data values. This formats the data properly before calling the HTTP request.

This update is more related to the JSON support PR #75 , the changes of which were accidentally included in this branch. If we merge #75 first as planned, this will be irrelevant.